app/views/tournaments/show.html.slim

```diff
@@ -35,9 +35,9 @@
               - unless @current_user_dropped
                 h5.card-title Register for this Event
                 = simple_form_for :player, url: tournament_players_path(@tournament) do |f|
-                  = render 'players/form', f: f, default_name: current_user ? current_user.nrdb_username : ''
+                  = render 'players/form', f: f, default_name: current_user.nrdb_username, new_self_registration: true
```

Comment:
why is new_self_registration added here as always true?

Reply:
Good question, I'll take a look.

Reply:
If you look at the tree of elements above this, there's `- if @tournament.self_registration?`, and under that there's `- if @current_user_player`, and this is in the else block of that, so new_self_registration will only be true if self registration is turned on and you're not already registered for the tournament.